compiler/damlc/daml-doc/src/DA/Daml/Doc/Transform/Instances.hs

```diff
@@ -75,3 +76,18 @@ distributeInstanceDocs opts docs =
             anchor <- ad_anchor ad
             Map.lookup anchor imap
         }
+
+    addIfaceImpls :: InstanceMap -> TemplateDoc -> TemplateDoc
+    addIfaceImpls imap td = td
+        { td_impls =
+            [ ImplDoc iface_type
+            | InstanceDoc {id_type} <-
+                maybe [] Set.toList $ do
+                  anchor <- td_anchor td
+                  Map.lookup anchor imap
+            , Just "Implements" == getTypeAppName id_type
```

Comment:
Can we restrict this to also check that the module of the typeclass is `DA.Internal.Desugar` here? Maybe by checking the anchor.

Reply:
You should be able to use `classAnchor` to generate the anchor to compare against `getTypeAppAnchor`.

Reply:
I added the module name to the `InstDoc` and now check we check for it here.